nix/ouroboros-network.nix

```diff
@@ -90,7 +90,7 @@ let
         preCheck =
           lib.mkForce
             (if buildSystem == "x86_64-linux"
-            then "export GHCRTS=-M300M"
+            then "export GHCRTS=-M400M"
```

Comment:
I removed the heap size limit increase from the PR, so this should be removed I think